### ISSUE TITLE
Correct the parameter used with the search form

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#628]
**Describe the bug**

In searchform.php, the parameter/argument that is used is named `$args['label']`, the correct name for use with 
get_search_form() is `$args['aria_label']` so this needs to be updated.
See https://developer.wordpress.org/reference/functions/get_search_form/#parameters

<del>The theme calls get_search_form() in 3 places, but the aria_label arg is not used.</del>

**To Reproduce**
1. View the markup of a search form, for example the search widget or the 404 page.
2. See that no ARIA label is used.

**Expected behavior**
Each form should have a relevant ARIA label, since it is possible to have more than one search form on the same page.

**Comment by carolinan** [comment]
Because this bug is also present in the Twenty Twenty theme, a trac ticket needs to be opened to solve the problem for that theme.

**Comment by aristath** [comment]
Using the `get_search_form` function will actually call the `searchform.php` file when it exists.
Since the theme already has that file, there is no need to pass any arguments to the function :+1:

**Comment by carolinan** [comment]
Then how do we add the unique labels?

**Comment by aristath** [comment]
You are correct, I misread the original post. My bad...
I'll assign this one to myself and take care of it now :+1:

**Comment by carolinan** [comment]
trac ticket created
